test/integration/OAuthClientsTest.php

```diff
@@ -36,18 +36,27 @@ function setUp(){
         $this->clientId = $personaConf['oauthClient'];
         $this->clientSecret = $personaConf['oauthSecret'];
 
-        $this->personaClientOAuthClient = new OAuthClients(array(
-            'persona_host' => $personaConf['host'],
-            'persona_oauth_route' => '/oauth/tokens'
-        ));
-        $this->personaClientUser = new Users(array(
-            'persona_host' => $personaConf['host'],
-            'persona_oauth_route' => '/oauth/tokens'
-        ));
-        $this->personaClientTokens = new Tokens(array(
-            'persona_host' => $personaConf['host'],
-            'persona_oauth_route' => '/oauth/tokens'
-        ));
+        $this->personaClientOAuthClient = new OAuthClients(
+            'unittest',
```

Comment:
If this is the scope belonging to the test user I think it needs to be more detailed for auditing purposes, using the [scope guidelines](https://github.com/talis/persona-server/wiki/A-note-on-scopes) e.g:

    integrationtest:testuser@personaphpclient